docs/elasticsearch.md

```diff
@@ -0,0 +1,204 @@
+# Elasticsearch
```

Comment:
Is there a reason why this is not in the `example` directory? IMO we should put examples in there. We can put the unit files in there so people can use them directly when checking out the repository. A README for each example explains then how it works. What do you guys think?